rtb/analisi-requisiti/src/casi-uso.d/utente-autenticato.tex

```diff
@@ -1 +1,278 @@
-C
+
+\textbf{UC3-Modifica account }
+\begin{itemize}
+    \item \textbf{Attore principale:} utente autenticato.
+    \item \textbf{Precondizioni:} l'utente è autenticato all'interno del sistema.
+    \item \textbf{Postcondizioni:} le modifiche fatte alle informazioni dell'account dell'utente sono
+    salvate dal sistema.
+    \item \textbf{Scenario principale:}
+        \begin{enumerate}
+            \item L'utente seleziona l'opzione di modifica dei dati del suo account;
+            \item L'utente può scegliere se modificare sia la password che l'email o solo una
+            di esse;
+            \item L'utente inserisce la nuova email o la nuova password o entrambe;
+            \item L'utente conferma le modifiche fatte;
+            \item Il sistema comunica all'utente che la modifica è avvenuta con successo.
+        \end{enumerate}
+\end{itemize}
+
+\textbf{UC4-Logout}
+\begin{itemize}
+    \item \textbf{Attore principale:} utente autenticato.
+    \item \textbf{Precondizioni:} l'utente è autenticato presso il sistema.
+    \item \textbf{Postcondizioni:} l'utente non è più autenticato all'interno del sistema.
+    \item \textbf{Scenario principale:}
+    \begin{enumerate}
+        \item L'utente seleziona l'opzione di logout;
+        \item Il sistema chiede all'utente la conferma della scelta;
+        \item L'utente conferma la scelta;
+        \item Il sistema re-indirizza l'utente alla home del sistema.
+    \end{enumerate}
+\end{itemize}
+
+\textbf{UC5-Creazione profilo}
+\begin{itemize}
+    \item \textbf{Attore principale:} utente autenticato.
+    \item \textbf{Precondizioni:} l'utente è connesso al sistema e sta visualizzando la lista dei suoi profili.
+    \item \textbf{Postcondizioni:} il profilo creato dall'utente, insieme a tutte le relative informazioni,
+    è salvato dal sistema nella lista dei profili dell'utente.
+    \item \textbf{Scenario principale:}
+    \begin{enumerate}
+        \item L'utente seleziona l'opzione di creazione di un nuovo profilo;
+        \item Il sistema chiede all'utente di scegliere se creare un profilo di tipo cliente
+        o ristoratore;
+        \item L'utente sceglie la tipologia di profilo;
+        \item L'utente inserisce i dati necessari per la creazione del profilo scelto;
+        \item L'utente sceglie l'opzione di conferma dei dati inseriti;
+        \item Il sistema comunica all'utente che la creazione del profilo è avvenuta con successo;
+        \item L'utente visualizza la lista dei suoi profili, dove è stato aggiunto il profilo appena creato.
+    \end{enumerate}
+    \item \textbf{Generalizzazioni:}
```

Comment:
Domanda stupida : la generalizzazione non è in realtà lo use case "padre"?

Reply:
vero, i figli dovrebbero essere "Specializzazioni"